playbook/app/pb_kits/playbook/plugins/pb_chart.js

```diff
@@ -67,10 +68,18 @@ class pbChart {
     }
   }
 
+  setupTheme() {
+    if (this.options.dark) {
+      Highcharts.setOptions(highchartsDarkTheme)
+    } else {
+      Highcharts.setOptions(highchartsTheme)
+    }
+  }
+
```

Comment:
Let's optimize this a bit..

```suggestion
  setupTheme({ dark }) {
    Highcharts.setOptions(dark ? highchartsDarkTheme : highchartsTheme)
  }

```

Reply:
See the following two suggestions as well

Reply:
This one will not work because dark is undefined.  But ternary will definitely work and yes we should optimize it.

Reply:
But this will work:
`setupTheme() {
    this.options.dark ?  Highcharts.setOptions(highchartsDarkTheme) : Highcharts.setOptions(highchartsTheme)
 }`

Reply:
You could default it

`({ dark=false })`

Reply:
I mean, we can access dark through options only, so we have to say if `this.options.dark` . If we just call `dark` it will be undefined.

Reply:
And I did default it in React. Rails just render the react class component.